src/main/scala/is/hail/methods/LDPrune.scala

```diff
@@ -209,7 +209,7 @@ object LDPrune {
 
         while (!done && qit.hasNext) {
           val (v2, bpv2) = qit.next()
-          if (v.contig != v2.contig || (v.start - v2.start) > windowSize)
+          if (v.contig != v2.contig || math.abs(v.start - v2.start) > windowSize)
```

Comment:
if it's guaranteed that v.start > v2.start, you shouldn't need math.abs.

maybe v and v2 aren't the best names.